README.md

```diff
@@ -16,3 +16,12 @@ npm start
 yarn install
 yarn start
 ```
+
+**Docker setup**
+```shell
+# Build container image from local filesystem with Node version as build arg
+docker build --build-arg NODE_VERSION=19 --no-cache -t website:<tag> -f Containerfile .
+
+# Run container image with mapping port 80 on your computer
+docker run -dit -p 80:8000 website:<tag>
```

Comment:
```suggestion
docker run -dit -p 8000:8000 website:<tag>
```
I understand why you default to 80 here and it also makes a lot of sense. What I think is that this might confuse folks since the console output for example says the following:
```
You can now view open-git-ops in the browser.
⠀
  Local:            http://localhost:8000/
```